lib/affirm/charge_event.rb

```diff
@@ -0,0 +1,20 @@
+module Affirm
+  class ChargeEvent
+    attr_reader :id, :transaction_id, :type, :created,
+      :amount, :fee, :order_id, :shipping_carrier, :shipping_confirmation,
+      :fee_refunded
+
+    def initialize(attrs)
+      @id                    = attrs["id"]
+      @transaction_id        = attrs["transaction_id"]
+      @type                  = attrs["type"]
+      @created               = attrs["created"]
+      @amount                = attrs["amount"]
+      @fee                   = attrs["fee"]
```

Comment:
I would normally prefer that the attrs are stored in an ivar with methods that act on the attrs. 

This also might be a good time to do any type conversion. (which is why above methods look cleaner). Is fee returned as a float?